docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -73,8 +74,8 @@ int main() {
 
             msgs_get.add_option(
                 dpp::command_option(dpp::co_integer, "quantity", "Quantity of messages to get. Max - 100.")
-                .set_max_value(100)
-                .set_min_value(1)
+                .set_max_value(100ll)
```

Comment:
this literally looks like `10011`... can we do anything about the readability of this?

Reply:
```cpp
const int64_t min_val{1};
const int64_t max_val{100};

...
.set_min_value(min_val).set_max_value(max_val)
```
perhaps this?

Reply:
It looks pretty different in docs. But you could also work around with a static_cast, however, this is probably better suited.
![image](https://github.com/brainboxdotcc/DPP/assets/46899449/729109b9-4937-4495-b8d4-f79a3d6c0b00)

Reply:
> ```c++
> const int64_t min_val{1};
> const int64_t max_val{100};
> 
> ...
> .set_min_value(min_val).set_max_value(max_val)
> ```
> 
> perhaps this?

This could work too yeah. I would prefer this.